tuna/rocmlir/rocmlir_tables.py

```diff
@@ -229,13 +290,155 @@ def get_query(self, sess, result_obj, session_id):
     self.logger.info("result query %s-%s", session_id, self.config)
     return query
 
-  # +++pf:  rewrite me for tuningRunner.py output!
   def parse(self, lines):
     """parse logger output line for result data """
     line = lines.splitlines()[-1]
     print(f"line being parsed is '{line}'", file=sys.stderr)
     return line.split('\t')
 
+  def export_as_tsv(self, filename, dbt, append=False):
```

Comment:
Write the relevant contents of the results table in the form that perfRunner.py expects.  For now, we're just dropping the Tuna process in as a replacement for our existing tuning process;  later, we can modify perfRunner.py to query the database directly.